R-package/tests/testthat/test_lgb.Booster.R

```diff
@@ -1341,7 +1341,7 @@ test_that("Booster's print, show, and summary work correctly", {
         .has_expected_content_for_fitted_model(log_txt)
 
         # summary()
-        log_text <- capture.output({
+        log_txt <- capture.output({
```

Comment:
This one was actually a bug! The assertions below this check a variable named `log_txt`, which is the output of the previous command. With this change, that assertion actually checks the result of this call.